### DISCUSSION TITLE
Bot disconnecting and reconnecting from voice channel repeatedly

### DISCUSSION THREAD

**Discussion by GustavoVieiraDrk** [discord-jda/JDA#2440] [Questions and Help] [answered]
For the last few days I have been facing the following problem, when the bot enters the voice channel after 5 seconds it starts disconnecting and reconnecting repeatedly. I believe this is happening because of this:

Discord api-annauncementes "Starting March 15, 2023, all apps with Voice Connections using IP Discovery must send 74-byte UDP packets. IP Discovery requests sending the deprecated 70-byte packet will no longer receive a response." 

more details see: https://discord.com/developers/docs/topics/voice-connections#ip-discovery

I'm new in the development of bots for discord, I've tried everything within what I know but I couldn't solve it, if anyone can help me, thank you in advance. I really don't know what to do.

Obs: Don't return any stacktraces errors.

**Comment by MinnDevelopment** [comment, marked as answer]
This has been fixed already. Here is the announcement from the [JDA discord](https://discord.gg/jda):

> Discord is rolling out a breaking change to audio connections today. **Everyone** who makes use of audio connections should update to the latest version of JDA. Affected bots will struggle connecting to channels, resulting in join/leave loops.
> 
> For people using v5 (recommended), update to **5.0.0-beta.6** or later.
> The fix has been backported to v4 (legacy) with **4.4.1_353**, however we recommend updating to v5 as soon as possible.

See https://canary.discord.com/channels/125227483518861312/169489002708074496/1087482646315159613

**Reply by GustavoVieiraDrk** [reply]
Thank you very much, I couldn't see the link due to the error below.

Error in open the link - https://user-images.githubusercontent.com/63684172/230800657-5e5a52b9-826a-44fc-a48c-c93892901f86.png

However, the excerpt from the quoted text was of great help.
I updated the JDA to 5.0.0-beta.6 and as far as I tested it worked fine.
